index.html

```diff
@@ -5,12 +5,102 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/846a6339d9.js" crossorigin="anonymous"></script>
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <section class="header-top-section">
+            <a href="#"><img class="logo" src="img/clipart822727.png" alt="Cupcake Logo"></a>
+            <section class="header-align">
+                <i class="fas fa-bars"></i>
```

Comment:
Could this be hidden when not in mobile view?

Reply:
If that is in reference to the hamburger icon on desktop it is hidden. just checked it.